src/celery-cmd

```diff
@@ -3,7 +3,7 @@
 # Luca Pasquali <luca.pasquali@geosolutionsgroup.com>
 CELERY_BIN=${CELERY_BIN:-"$(which celery||echo celery)"}
 CELERY_APP=${CELERY_APP:-"geonode.celery_app:app"}
-CELERY__STATE_DB=${CELERY__STATE_DB:-"/mnt/volumes/statics/worker.state"}
+CELERY__STATE_DB=${CELERY__STATE_DB:-"/mnt/volumes/statics/worker@%h.state"}
```

Comment:
Is this a typo?

Reply:
No, it isn't. It's a way to have the worker state file numbered automatically. This is required in case you have multiple Celery instances.